plugins/ImageGraph/API.php

```diff
@@ -493,6 +494,8 @@ public function get(
 
             // render graph
             $graph->renderGraph();
+        } catch (InvalidDimensionException $e) {
+            throw new Exception('Error: the graph dimension is not valid. Please try larger width and height values or use 0 for default values.');
```

Comment:
Any reason for not simply re-throwing the caught exception and setting the message where the exception actually occurs?

Reply:
You're right, that's better solution 👍 , I'll make it like that.